$NetBSD: patch-ae,v 1.4 2001/04/08 16:25:28 dmcmahill Exp $

libtool doesn't like either f77 or f2c-f77 without a tag.
also link in fortran libs

--- config/Makeso.incl.in.orig	Tue Mar 13 03:53:19 2001
+++ config/Makeso.incl.in	Fri Mar 30 06:10:34 2001
@@ -19,4 +19,5 @@
 LDFLAGS = @LDFLAGS@
 LIBS = @LIBS@
+FLIBS = @FLIBS@
 
 #******************************************************************* 
@@ -33,7 +34,7 @@
 
 LTCOMPILE = $(LIBTOOL) --mode=compile $(QUIET) $(CC) $(INCLUDES) $(CPPFLAGS) $(CFLAGS)
-FLTCOMPILE = $(LIBTOOL) --mode=compile $(QUIET) $(FC) $(INCLUDES) $(FFLAGS)
+FLTCOMPILE = $(LIBTOOL) --mode=compile --tag=CC $(QUIET) $(FC) $(INCLUDES) $(FFLAGS)
 
-LINK = $(LIBTOOL) --mode=link $(QUIET) $(CC) $(CFLAGS) $(LDFLAGS) -o $@
+LINK = $(LIBTOOL) --mode=link $(QUIET) $(CC) $(CFLAGS) $(LDFLAGS) $(FLIBS) -o $@
 
 OBJECTS = $(OBJS:.o=.lo)
